### DISCUSSION TITLE
« freerouting » is failed. Eject disk image on Mac OS (freerouting-1.6.5-macos-x64.dmg)

### DISCUSSION THREAD

**Discussion by Cyrillinux** [freerouting/freerouting#166] [General]
Hello,
I can't throw Freerouting under MacOS.
It's for my all students same thing. (We use kicad, under MacOS El capitan (old Mac) and Mac OS Monterey (new Mac)).
Many thanks.
Cyril

**Comment by andrasfuchs** [comment]
Hi there,

Sorry for the inconvenience! Mac is not supported at the moment, because we don't have a developer that have Apple Developer License.

Still, it should work on Mac, if you use it as a command line tool while you have Java 17 or 18 installed. Keep in mind that Java 19 doesn't work.

So, what I think you need to do is:
1. Install Java 17 on the machines where you would like to use Freerouting. You can download it from here: https://adoptium.net/temurin/releases/
2. You have to options now: you can use KiCad and install it as a plugin or you can download the command line tool version.
3. The command line tool version is here: https://github.com/freerouting/freerouting/releases . You need to download the `.jar` file, currently it's `freerouting-1.6.5.jar`.
4. At this point you can run the software on your system by `java -jar freerouting-1.6.5.jar`. ([see this tutorial for details](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Run-JAR-file-example-windows-linux-ubuntu))

I hope it helps!
Andras
